bedrock/mozorg/templates/mozorg/about/history-details.html

```diff
@@ -17,7 +17,7 @@ <h1 class="title-shadow-box">{{ _('History of the Mozilla Project') }}</h1>
       {% trans coderush='http://clickmovement.org/coderush',
                sourcerelease='http://web.archive.org/web/20021001071727/wp.netscape.com/newsref/pr/newsrelease558.html'
       %}
-        The Mozilla project was <a href="{{ coderush }}">created in 1998</a>
+        The Mozilla project was created in 1998
```

Comment:
There's no need for a new string; this can be a simple change: code rush is available at https://air.mozilla.org/code-rush/